src/db/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "ugbio_db"
+version = "1.1.0.dev0"
+requires-python = ">= 3.11"
+dependencies = [
+    "ugbio_core",
```

Comment:
You don't need the core at all. I think once you remove it you will need to add pandas. But that is it.

Reply:
Fixed